packages/component-library-react/src/Calendar/CalendarNavigationButtons.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+interface CalendarNavigationButtonsProps
+  extends React.DetailedHTMLProps<React.ButtonHTMLAttributes<HTMLButtonElement>, HTMLButtonElement> {
+  onPreviousClick?: React.MouseEventHandler<HTMLButtonElement>;
+  onNextClick?: React.MouseEventHandler<HTMLButtonElement>;
+  previousIcon?: any;
+  nextIcon?: any;
+  previousTitle?: string;
+  nextTitle?: string;
+  children?: React.ReactNode;
+}
+
+export const CalendarNavigationButtons: React.FC<CalendarNavigationButtonsProps> = ({
+  onPreviousClick,
+  onNextClick,
+  previousIcon,
+  nextIcon,
+  children,
+  ...props
+}) => (
+  <div className="utrecht-calendar__navigation-buttons">
+    <button className="utrecht-calendar__navigation-buttons-item" onClick={onPreviousClick} {...props}>
```

Comment:
How about `navigation-buttons-item` ➝ `navigation-button`?

Reply:
Do you mean skip the parent (`utrecht-calendar__navigation-buttons`)?